client/my-sites/earn/style.scss

```diff
@@ -71,10 +71,10 @@
 	}
 }
 
-.earn .promo-section__promos .gridicon {
-	fill: var(--color-primary-10);
-	position: relative;
-	top: -8px;
+.earn .promo-card {
+	@include breakpoint-deprecated( ">1280px" ) {
+		width: calc(33% - 1em);
+	}
```

Comment:
The three column width is only be applied on the earn page, not as part of the general component. We could easily move this to the component. Three columns is tight, and I wasn't sure it would fit with spacing/card content in other places.